subsys/bluetooth/shell/bt.c

```diff
@@ -1377,7 +1377,11 @@ static int cmd_per_adv_data(const struct shell *shell, size_t argc,
 	return 0;
 }
 #endif /* CONFIG_BT_PER_ADV */
+#endif /* CONFIG_BT_EXT_ADV */
+#endif /* CONFIG_BT_BROADCASTER */
 
+#if defined(CONFIG_BT_OBSERVER)
+#if defined(CONFIG_BT_EXT_ADV)
 #if defined(CONFIG_BT_PER_ADV_SYNC)
```

Comment:
`BT_PER_ADV_SYNC` depends on `BT_OBSERVER` and cannot be set if `BT_EXT_ADV` isn't set. In that sense, I suppose we could reduce this to simply be
```
#if defined(CONFIG_BT_PER_ADV_SYNC)
```